src/locale/az.js

```diff
@@ -0,0 +1,14 @@
+export default {
+    days: ['Bazar', 'Bazar ertəsi', 'Çərşənbə axşamı', 'Çərşənbə', 'Cümə axşamı', 'Cümə', 'Şənbə'],
+    daysShort: ['B.', 'B.E', 'Ç.A', 'Ç.', 'C.A', 'C.', 'Ş.'],
+    daysMin: ['B.', 'B.E', 'Ç.A', 'Ç.', 'C.A', 'C.', 'Ş.'],
```

Comment:
I believe "Bazar" (which is "Sunday" in Azerbaijani) shouldn't be the first element of array. It is last day of week in Azerbaijan.

Reply:
> I believe "Bazar" (which is "Sunday" in Azerbaijani) shouldn't be the first element of array. It is last day of week in Azerbaijan.

Indeed it is the last day of the week. However, if you read the code properly, you will see there is an option to set the first day of the week named `firstDay`. It is set to 1.
Basically this option will show the second element of the array as the first day in the Calendar UI which is "Bazar ertəsi".

Thanks for your comment!

Reply:
I didn't read the code, thanks for noticing!

Reply:
> I didn't read the code, thanks for noticing!

Please do not request any changes without properly reviewing the code and unnecessarily. Good luck!